analysis/pyproject.toml

```diff
@@ -5,10 +5,12 @@ description = ""
 authors = ["Serokell <hi@serokell.io>"]
 
 [tool.poetry.dependencies]
-python = "3.9.*"
+python = ">=3.7.1,<3.10"
 numpy = "^1.20.2"
 scipy = "^1.6.2"
 sklearn = "^0.0"
+pytest = "^6.2.3"
```

Comment:
In the second half of [this thread](https://serokell.slack.com/archives/C01F8AQSHGE/p1617113349023000) people told me that test dependencies should go to `dev-dependencies` rather than `dependencies` , so shouldn't `pytest` be moved there?

Reply:
Yeah, seem like a good idea, will move it. Tbh I just haven't checked as PyCharm edits poetry stuff it on its own.